assembly/src/etc/tornado.properties

```diff
@@ -1,8 +1,35 @@
 tornado.opencl.print = false
 tornado.debug = false
-tornado.platform = 0
+#
+# Index of the default driver (OpenCL vs PTX) used for TaskSchedule.
+# Please check output of "tornado --devices" for the correct value.
+tornado.driver = 0
+#
+# Index of the default default device within default driver 
+# used for TaskSchedule.
+# Please check output of "tornado --devices" for the correct value.
 tornado.device = 0
 tornado.events.log = false
 tornado.vectors.enable = true
 tornado.kernels.parallelise = true
 tornado.opencl.gpu.block.x = 512
+#
+# Index of the default platform (like AMD, Intel, NVIDIA etc)
+# used in OpenCL-specific applications.
+# Please note that this property does not affect default driver/device
+# settings for TaskSchedule.
+tornado.opencl.platform = 0
+#
+# Index of the default device withing default platform
+# used in OpenCL-specific applications.
+# Please note that this property does not affect default driver/device
+# settings for TaskSchedule.
+tornado.opencl.device = 0
+#
+# Index of the default device used in PTX-specific applications.
```

Comment:
Index of the default PTX device. 
There is no specific setting corresponding to the PTX platform (unlike the OpenCL). 
This is because the PTX backend has a single platform. All GPUs are groups into the same platform.
Note that this property does not affect the default `driver/device` setting for `TaskSchedule`s.